pom.xml

```diff
@@ -35,6 +35,8 @@
     <packaging>pom</packaging>
 
     <properties>
+        <asciidoctor.plugin.version>1.5.8</asciidoctor.plugin.version>
```

Comment:
@ahus1 We can at least use the 1.5.8, right?

Reply:
@mabartos - yes, 1.5.8 is fine.